.gitignore

```diff
@@ -12,3 +12,4 @@ coverage.txt
 /.bin
 /.cache
 /bin
+/.idea
```

Comment:
This should't be a part of the this repo. Add IDE excludes to your [global gitignore](https://git-scm.com/docs/gitignore#_configuration).